python_modules/libraries/dagster-aws/dagster_aws/pipes.py

```diff
@@ -157,10 +169,22 @@ def no_messages_debug_text(self) -> str:
         )
 
 
+class CloudWatchEvent(TypedDict):
+    timestamp: int
+    message: str
+    ingestionTime: int
+
+
 @experimental
 class PipesCloudWatchMessageReader(PipesMessageReader):
     """Message reader that consumes AWS CloudWatch logs to read pipes messages."""
 
+    def __init__(self, client: Optional[boto3.client] = None):
+        """Args:
+        client (boto3.client): boto3 CloudWatch client.
+        """
+        self.client = client or boto3.client("logs")
```

Comment:
I decided to automatically create default aws clients for the pipes components. 
AWS clients are usually configured via environment variables and local configs, so I think that's a sensible default. 

Is this ok to do?

Reply:
Seems quite sensible! 👍🏻